app/views/aliquots/_well_failing_aliquot.html.erb

```diff
@@ -1,6 +1,18 @@
-<%= presenter.form.check_box(id) if failable?(container) %>
+<%= presenter.form.check_box(id, disabled: !failable?(container), data: {
+  quadrant_index: container.quadrant_index,
+  prevent_quadrant_fail: prevent_quadrant_fail?(container)
+}) %>
 <%= presenter.form.label id do %>
-  <div id="aliquot_<%= id %>" class="aliquot <%= permanent_state(container) %>" data-well="<%= id %>">
-    <%= id %>
+  <div id="aliquot_<%= id %>"
+       class="aliquot <%= container.state %>"
+       data-toggle="tooltip"
+       data-placement="top"
+       title="<%= id %>"
+       >
+    <% if container.respond_to?(:contains_control?) && container.contains_control? %>
+      <div class="control-type">
+        <%= container.control_info_formatted %>
+      </div>
+    <%end%>
```

Comment:
Moves the well name into a tooltip, as things were a bit cramped on 384 well plates. This also lets us instead show the control state for a well.

And I've just worked out why the standard aliquot template needed the `container.respond_to?(:contains_control?)`! It's because it's not exclusively used by the presenters, so sometimes gets V1 wells.